pydantic_ai_slim/pydantic_ai/agent.py

```diff
@@ -1110,3 +1115,6 @@ class _MarkFinalResult(Generic[ResultData]):
     """
 
     data: ResultData
+    """The final result data."""
+    tool_name: str | None
+    """Name of the final result tool, None if the result is a string."""
```

Comment:
```suggestion
    final_tool_name: str | None
    """Name of the final result tool, None if the result is a string."""
```

And elsewhere, just for clarity?

Reply:
I think that's tautologous with the dc name.

Reply:
Ah yes, I missed that.